packages/concerto-core/test/data/parser/mapdeclaration/mapdeclaration.goodkey.declaration.event.cto

```diff
@@ -0,0 +1,22 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace com.acme@1.0.0
+
+event Activity {}
+
+map Dictionary {
+  o Activity
+  o String
+}
```

Comment:
This is an interesting idea, not necessarily wrong. But not in the design.

Presumably, we would key-off of the timestamp of the event?

Reply:
Yes this should not have been committed, and it is not referenced from any test case.